tests/InferTypesTest.elm

```diff
@@ -231,3 +276,52 @@ niceVarName =
             , ( 259, "z9" )
             , ( 260, "a10" )
             ]
+
+
+isParametric : Test
+isParametric =
+    let
+        runTest : ( Type, Bool ) -> Test
+        runTest ( input, output ) =
+            test (dumpType input) <|
+                \() ->
+                    input
+                        |> Type.isParametric
+                        |> Expect.equal output
+    in
+    describe "Type.isParametric" <|
+        List.map runTest
+            [ ( Unit, False )
+            , ( Bool, False )
+            , ( Char, False )
+            , ( Int, False )
+            , ( String, False )
+            , ( Var 0, True )
+            , ( Function Int String, False )
+            , ( Function (Var 0) Int, True )
+            , ( Function String (Var 0), True )
+            , ( Function Int (Function (Var 0) (Var 0)), True )
+            , ( List Int, False )
+            , ( List (Var 0), True )
+            , ( List (List Int), False )
+            , ( List (List (Var 0)), True )
+            , ( Tuple Int String, False )
+            , ( Tuple (Var 0) Int, True )
+            , ( Tuple String (Var 0), True )
+            , ( Tuple (List (Var 0)) Int, True )
+            , ( Tuple Char (Tuple Int (Var 0)), True )
+            , ( Tuple3 Int String Bool, False )
+            , ( Tuple3 (Var 0) Int Char, True )
+            , ( Tuple3 String (Var 0) Unit, True )
+            , ( Tuple3 Bool Unit (Var 0), True )
+            , ( Tuple3 (List (Var 0)) Int Char, True )
+            , ( Tuple3 String (Function (Var 0) Int) Unit, True )
+            , ( Tuple3 Bool Unit (Tuple (Var 0) Int), True )
+            ]
+
+
+dumpType : Type -> String
+dumpType type_ =
+    type_
+        |> Type.toString Type.emptyState
+        |> Tuple.first
```

Comment:
There is `dumpType` in both InferTypesFuzz and InferTypesTest - we can move it to TestHelpers